### PR TITLE
Changes how butchering works, Changes butchering drops

### LINKED ISSUE #8963: Pete no longer provides meat when butchered. Other animals too.
Pete and the Fox no longer provide meat when butchered.

### PR DESCRIPTION
**Pull request by Firecage**
This PR changes how butchering works. So instead of a skin_type and meat_type(and meat_amount), it is a list of items, with their amounts. So it is actually easier to add custom stuff to simple animals for when you butcher them.

This also changes some of the meat that some simple animals drops.

Cats: Changed from 3 to 2.
Crabs: Changed from 2 to 1.
Fox: Can now butcher for meat again.
Xenos: Changed from 3 to 4.
Xeno Empress: Is now 10 meat, and 2 hides.
Bear: Changed from 3 to 5.
Killer Tomato: Changed from 1 to 2.
Spiders: Spider Nurses now also gives meat, and spider legs is standardized to always be 8.
Bats: Can now give meat aswell.

Tested and everything works as intended.

Also Fixes #8963 

PS: Thanks Comic for helping me.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 1]
```diff
@@ -369,8 +366,6 @@
 				harvest(user)
 			return
 
-	..()
```
Comment (phil235) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: this is very much needed, you won't be able to attack animals with items if you remove that.

**Comment by Firecage** [comment]
Fuck, didn't actually notice I removed that.

**Comment by Dorsisdwarf** [comment]
Xenoes should have 4 meat again, since it's hard enough to get them, might as well reward the Danger Chef

**Comment by Firecage** [comment]
@Dorsisdwarf Thats what I did...>.> Xeno Empress has 10 though

**Pull request merged**